WordPress/Classes/Services/Store.swift

```diff
@@ -22,53 +22,162 @@ class StoreKitTransactionObserver: NSObject, SKPaymentTransactionObserver {
 // stored property since it's a generic class.
 struct StoreKitCoordinator {
     static let instance = StoreCoordinator(store: StoreKitStore())
+    
+    static let TransactionDidFinishNotification = "StoreCoordinatorTransactionDidFinishNotification"
+    static let TransactionDidFailNotification   = "StoreCoordinatorTransactionDidFailNotification"
+    static let NotificationProductIdentifierKey = "StoreCoordinatorNotificationProductIdentifierKey"
 }
 
+typealias PendingPayment = (planID: PlanID, productID: String, siteID: Int)
```

Comment:
Thanks, this makes a lot of sense. I've updated things to only store the product ID (SKU, and we now purchase products instead of plans.